modal/app.py

```diff
@@ -1094,6 +1097,41 @@ def main():
 
         return self
 
+    async def set_tags(self, tags: Mapping[str, str], *, client: Optional[_Client] = None) -> None:
```

Comment:
See discussion on slack about whether we want `client: ...` in here long term, but I think we need it for now.